public/_locales/en/messages.json

```diff
@@ -1043,6 +1043,9 @@
   "recommended": {
     "message": "Recommended"
   },
+  "helpUkraine": {
+    "message": "Help Ukraine "
```

Comment:
Adding 'Ukrainian flag' emoji here instead of https://github.com/madfish-solutions/templewallet-extension/pull/899/files#diff-6786c07c59f2cf920a375e9968219f73a9d91fde48feff2f5fdd88d4337fe11cR214 will make your code simpler